### ISSUE TITLE
Tabula not displaying PDF content

### ISSUE THREAD

**Issue by HEtothe** [tabulapdf/tabula#538]
1. I upload a PDF on the My Files page
2. I click "Extract data"
3. At this point, I should be given a view of the PDF contents so that I can pick what to extract. But all I get is a blank gray box. I can create a red selector box on that blank gray box; and then when I click through to preview the selected table, there is content. But I cannot see what I am selecting!

Can anyone help?

**Comment by HEtothe** [comment]
Just realised I didn't give the problem document - I think its shape may have something to do with the problem.

[Legal Week 110816 - Top50_LLP_SW_SMALLEST1.pdf](https://github.com/tabulapdf/tabula/files/420051/Legal.Week.110816.-.Top50_LLP_SW_SMALLEST1.pdf)

**Comment by jeremybmerrill** [comment]
I don't think it's the shape, but there is something weird going on with the PDF. I'm getting errors I've never gotten before (from JPedal, I think, which is what we use for rendering PDFs to an image):

```
Sep 17, 2016 11:58:14 AM org.apache.pdfbox.pdmodel.font.PDTrueTypeFont getawtFont
INFO: Can't read the embedded font CWMJZM+Times-Roman
Sep 17, 2016 11:58:14 AM org.apache.pdfbox.pdmodel.font.PDTrueTypeFont getawtFont
INFO: Using font Serif.plain instead
Sep 17, 2016 11:58:14 AM org.apache.pdfbox.pdmodel.graphics.xobject.PDPixelMap getRGBImage
SEVERE: Something went wrong ... the pixelmap doesn't contain any data.
Sep 17, 2016 11:58:14 AM org.apache.pdfbox.util.operator.pagedrawer.Invoke process
WARNING: getRGBImage returned NULL
Sep 17, 2016 11:58:14 AM org.apache.pdfbox.pdmodel.graphics.xobject.PDPixelMap getRGBImage
SEVERE: Something went wrong ... the pixelmap doesn't contain any data.
Sep 17, 2016 11:58:14 AM org.apache.pdfbox.util.operator.pagedrawer.Invoke process
WARNING: getRGBImage returned NULL
Sep 17, 2016 11:58:16 AM org.apache.pdfbox.pdmodel.graphics.xobject.PDPixelMap getRGBImage
SEVERE: Something went wrong ... the pixelmap doesn't contain any data.
Sep 17, 2016 11:58:16 AM org.apache.pdfbox.util.operator.pagedrawer.Invoke process
WARNING: getRGBImage returned NULL
Sep 17, 2016 11:58:16 AM org.apache.pdfbox.pdmodel.graphics.xobject.PDPixelMap getRGBImage
SEVERE: Something went wrong ... the pixelmap doesn't contain any data.
Sep 17, 2016 11:58:16 AM org.apache.pdfbox.util.operator.pagedrawer.Invoke process
WARNING: getRGBImage returned NULL
```

**Comment by kbduvall** [comment]
I'm getting the same issue.. only all white previews. I can still create a selection block on the blank preview, and their is still content as HEtothe mentioned. 

I'm using v1.1.0, the OS X App bundle version.

Here is a link to the PDF http://www.cisco.com/c/en/us/products/collateral/video/traditional-size-taps/datasheet-c78-732058.pdf

**Comment by jeremybmerrill** [comment]
Error from kbduvall's PDF looks like this:

````
WARNING: java.lang.NullPointerException
java.lang.NullPointerException
	at technology.tabula.ObjectExtractor.fillPath(ObjectExtractor.java:283)
	at org.apache.pdfbox.util.operator.pagedrawer.FillEvenOddRule.process(FillEvenOddRule.java:57)
	at org.apache.pdfbox.util.PDFStreamEngine.processOperator(PDFStreamEngine.java:562)
	at org.apache.pdfbox.util.PDFStreamEngine.processSubStream(PDFStreamEngine.java:269)
	at org.apache.pdfbox.util.PDFStreamEngine.processSubStream(PDFStreamEngine.java:236)
	at org.apache.pdfbox.util.PDFStreamEngine.processStream(PDFStreamEngine.java:216)
	at technology.tabula.ObjectExtractor.drawPage(ObjectExtractor.java:153)
	at technology.tabula.ObjectExtractor.extractPage(ObjectExtractor.java:108)
	at technology.tabula.PageIterator.next(PageIterator.java:29)
	at technology.tabula.PageIterator.next(PageIterator.java:6)
````

so probably the same as #577 -- which I believe is likely to be a PDFBox issue, which will be fixed by upgrading to PDFBox 2.0, which is in progress at https://github.com/tabulapdf/tabula-java/tree/pdfbox2.0.

@jazzido: If my diagnosis is wrong, please reopen.
